src/styles/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  background-color: white;
```

Comment:
Use HEX instead of color names cause colors from color names are different in each browser.